src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -292,7 +294,7 @@ export const TradeInput = memo(({ isCompact }: TradeInputProps) => {
 
   const isLoading = useMemo(
     () =>
-      !isAnyTradeQuoteLoaded ||
+      (!isAnyTradeQuoteLoaded && !isTradeQuoteRequestAborted) ||
```

Comment:
If we have no quotes, but the trade quote request is aborted, we aren't actually still loading.